redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/requests/SslConfigurationRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.requests;
+
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sequenceiq.redbeams.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+
+@ApiModel(description = ModelDescriptions.SSL_CONFIGURATION_REQUEST)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SslConfigurationRequest {
```

Comment:
I suggest renaming this to `SslConfigurationV4Request` to be consistent with other request classes.